### DISCUSSION TITLE
mkdocs-material/blog/2022/chinese-search-support/

### DISCUSSION THREAD

**Discussion by giscus** [squidfunk/mkdocs-material#3888] [_]
# mkdocs-material/blog/2022/chinese-search-support/

Insiders adds Chinese language support for the built-in search plugin – a  feature that has been requested many times

https://squidfunk.github.io/mkdocs-material/blog/2022/chinese-search-support/

**Comment by samzong** [comment]
wonderful for chiese gays

**Reply by SakurajimaMaii** [reply]
Oh，my  friend, it should be chinese not chiese.😓

**Reply by samzong** [reply]
yeah , toooo excited 🤣

**Comment by nmnmcc** [comment]
Thanks！

**Comment by shushenghong** [comment]
我这好像还不行，环境如下
```
Name: mkdocs
Version: 1.3.0

Name: mkdocs-material
Version: 8.3.9

Name: jieba
Version: 0.42.1
```

mkdocs.yml也配置了
```
plugins:
  - search:
      separator: '[\s\u200b\-]'

```

但还是搜不了中文，不知道哪里有问题

**Reply by squidfunk** [reply]
It seems you are not using [Insiders](https://squidfunk.github.io/mkdocs-material/insiders/). Improved Chinese search support is currently only available for sponsors.

**Comment by dianweili** [comment]
While I want to ask one non-relative question, does material or mkdocs support proxy?
I use '-privacy' plugin, but I must connect Internet through http proxy. My proxy is ok on Chrome, but it doesn't work  on Powershell, on which I run mkdocs with material.

**Reply by dianweili** [reply]
I open '-privacy' plugin, and first I build  my site on my PC, after with I would move my site to out internal web, on which the site is deployed.

**Reply by nmnmcc** [reply]
Try to use tun mode, which fixes the problem urgently.

**Reply by dianweili** [reply]
> Try to use tun mode, which fixes the problem urgently.

Is this tun mode of mkdocs, or of material? Any reference? Thanks alot.

**Reply by nmnmcc** [reply]
> > Try to use tun mode, which fixes the problem urgently.
> 
> Is this tun mode of mkdocs, or of material? Any reference? Thanks alot.

https://en.m.wikipedia.org/wiki/TUN/TAP

**Comment by jasonren0403** [comment]
That's amazing! Thank you for the support!
